.gitlab/functional_test/system_probe.yml

```diff
@@ -83,6 +83,5 @@ kitchen_test_system_probe_windows_x64:
     - bash -l tasks/kitchen_setup.sh
   script:
     - bash -l tasks/run-test-kitchen.sh windows-sysprobe-test $AGENT_MAJOR_VERSION
-  # FIXME: temporarily allow this job to fail until we fix it
-  allow_failure: true
+  allow_failure: false
```

Comment:
You can remove that line, the default value of `allow_failure` is `false`.
Also, could you also remove the `retry: 0` on the next line? This was added to prevent Gitlab from retrying a job we knew would fail, so that isn't necessary anymore.